corehq/motech/repeaters/tests/test_repeater.py

```diff
@@ -1082,7 +1083,8 @@ def tearDown(self):
 
     def test_trigger_when_paused(self):
         # not paused
-        with patch.object(RepeatRecord, 'fire') as mock_fire:
+        with patch.object(RepeatRecord, 'fire') as mock_fire, \
```

Comment:
Can we remove patching `RepeatRecord.fire` since now only `SQLRepeatRecord.fire` would be used?